Library/Homebrew/cmd/update.sh

```diff
@@ -352,7 +352,11 @@ homebrew-update() {
         HOMEBREW_SIMULATE_FROM_CURRENT_BRANCH=1
         ;;
       --auto-update) export HOMEBREW_UPDATE_AUTO=1 ;;
-      --*) ;;
+      --*)
+        onoe "Unknown option: --${option}"
```

Comment:
`${option}` should already contain the leading dashes so:

```suggestion
        onoe "Unknown option: ${option}"
```

Reply:
Whoops, sorry, that was a thinko. 😪